examples/src/components/Error.jsx

```diff
@@ -23,8 +25,7 @@ export const Error = ({ error }) => (
             }}
         >
             <div className="gd-message-text">
-                <strong>{error.status}</strong>
-                { error.message ? <div>{error.message}</div> : null }
+                <strong>{JSON.stringify(error)}</strong>
```

Comment:
Sometimes the error is `{status: string}`, sometimes `{status {}}` - see `/basic-components`

It should be fixed in react-components, or at least JIRA task created.

Reply:
RAIL-670